src/util/specific.py

```diff
@@ -242,3 +251,34 @@ def translation2jsonb(cr, *fields):
         parallel_execute(cr, migrate_queries)
     parallel_execute(cr, all_cleanup_queries)
     return fields
+
+
+def create_backup_column(cr, table, column):
```

Comment:
This may be a better name, since sometimes we need to copy the values temporarily for some reason, but not necessarily as a "backup"
```suggestion
def create_copy_column(cr, table, column):
```

Reply:
Agree and Updated :saluting_face:

Reply:
Thank you for your great suggestion.
It's moved to `pg.py`.